<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindo</class>
 <widget class="QMainWindow" name="MainWindo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Router Command Executor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(179, 182, 215);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelImagen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>600</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img_topologia.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>349</y>
      <width>631</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e6e7f4;
    border: 2px solid #5d6899;
    border-radius: 10px;

    color: #4a4e7a;
    font-family: Arial;
    font-size: 12pt;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>627</width>
       <height>227</height>
      </rect>
     </property>
     <widget class="QLabel" name="labelResultados">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e6e7f4;
    border: 2px solid #5d6899;
    border-radius: 10px;

    color: #4a4e7a;
    font-family: Arial;
    font-size: 12pt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelUsuario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #5d6899;
    color: #e6e7f4;
    border-top-left-radius: 10px;
    border-bottom-left-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Usuario:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #5d6899;
    color: #e6e7f4;
    border-top-left-radius: 10px;
    border-bottom-left-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDispositivo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #5d6899;
    color: #e6e7f4;
    border-top-left-radius: 10px;
    border-bottom-left-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Dispositivo:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelComando">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #5d6899;
    color: #e6e7f4;
    border-top-left-radius: 10px;
    border-bottom-left-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Comando:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputUsuario">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: #e6e7f4;
    color: #4a4e7a;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
    font-size: 7pt;
    font-weight: bold;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPassword">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: #e6e7f4;
    color: #4a4e7a;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
    font-size: 7pt;
    font-weight: bold;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDispositivo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: #e6e7f4;
    color: #4a4e7a;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
    font-size: 7pt;
    font-weight: bold;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxComando">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: #e6e7f4;
    color: #4a4e7a;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
    font-size: 7pt;
    font-weight: bold;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEjecutar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: #4a4e7a;
    color: #e6e7f4;
    font-size: 9pt;
    font-weight: bold;
    border: none;
    padding: 5px 10px;
    border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Ejecutar</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameStatus">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>341</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imagen_topologia.qrc"/>
 </resources>
 <connections/>
</ui>
